src/java.base/share/classes/java/lang/foreign/MemoryLayout.java

```diff
@@ -275,23 +274,10 @@ default long byteOffset(PathElement... elements) {
      * in {@code elements} is {@code null}.
      */
     default long bitOffset(PathElement... elements) {
-        return computePathOp(LayoutPath.rootPath(this), LayoutPath::offset,
-                EnumSet.of(PathKind.SEQUENCE_ELEMENT, PathKind.SEQUENCE_RANGE), elements);
-    }
-
-    private static <Z> Z computePathOp(LayoutPath path,
```

Comment:
I'm generally not too sure about this. Yes, if there's a lot of code it might be useful to move it out - but in this case it's a single method used in several other methods in this interface. Why is it bad to use a private static interface method for something like this?